models/LineCont.yaml

```diff
@@ -0,0 +1,7 @@
+type: LineCont  # comment or string or block
+start: String
+end: String  # implicitly end of line.
```

Comment:
end is always `\n` in your examples so that is also redundant.

Notice the comment here that you copied, which says it is implicit, yet your examples explicitly add `\n`.

Reply:
fixed